pkg/source/mtadapter/adapter.go

```diff
@@ -241,14 +249,21 @@ func (a *Adapter) Update(ctx context.Context, obj *v1beta1.KafkaSource) {
 	}
 
 	ctx, cancelFn := context.WithCancel(ctx)
+
+	cancel = cancelContext{
+		fn:      cancelFn,
+		stopped: make(chan bool),
```

Comment:
Why do we want to name this `stopped`, since we put true into this channel after adapter starts, I think we should rename it to `inProgress` indicating adapter is currently running.

Reply:
adapter.Start() is blocking. We put true when the adapter stopped.

Reply:
I see. Ok, it should be named stopped.